lib/Backend/Lower.cpp

```diff
@@ -12510,6 +12514,49 @@ Lowerer::LowerBailOnNotObject(IR::Instr       *instr,
     return prevInstr;
 }
 
+IR::Instr *
+Lowerer::LowerCheckFunctionEntryPoint(IR::Instr *instr)
```

Comment:
>LowerCheckFunctionEntryPoint [](start = 9, length = 28)

you'll also need to emit a function body check once you start emitting `CheckFunctionEntryPoint` in `Inline::PrepareInsertionPoint`